### ISSUE TITLE
Bug: Default wallpaper flash

### ISSUE THREAD

**Issue by kirkas** [dpayne/wallock#10]
When locking and unlocking, the default system wallpaper (Anime girl on Hyprland) quickly flashes before the wallock takes over:

https://github.com/dpayne/wallock/assets/1159783/27bd2756-dae9-43b2-98da-8d5d59b5f0f0

**Comment by dpayne** [comment]
why not unset/disable the wallpaper?

**Comment by kirkas** [comment]
when I disable hyprland wallpaper with

```
disable_hyprland_logo = true
```

then it flashes with black background:



https://github.com/dpayne/wallock/assets/1159783/e95a024c-2334-47e7-a301-0fe91bb5f3eb

**Comment by dpayne** [comment]
yeah that's sort of unavoidable. I think there's a way to fix it on non-nvidia but I couldn't get it working on nvidia without deadlocking some of the time.

**Comment by dpayne** [comment]
Adding two paths (nvidia and non-nvidia) would involve a decent amount work and maintenance since the two flows would be very different. My hope is that after explicit sync is working I can try the single fast path again and see if that still works.
